api/lib/domain/usecases/create-certification-center-membership-by-email.js

```diff
@@ -0,0 +1,22 @@
+const { AlreadyExistingEntityError } = require('../errors');
+
+module.exports = async function createCertificationCenterMembershipByEmail({
+  certificationCenterId,
+  email,
+  certificationCenterMembershipRepository,
+  userRepository,
+}) {
+  const user = await userRepository.getByEmail(email);
+  const userId = user.id;
+
+  const isMembershipExisting = await certificationCenterMembershipRepository.isMemberOfCertificationCenter(
```

Comment:
**chipotage**
Je ne suis pas trop pour préfixer les booléen par `is`, car ça donne faussement l'impression d'une [voie progressive (be  + ing)](https://academicguides.waldenu.edu/writingcenter/grammar/verbforms) qui n'est pas justifié du point de vue grammatical.

Je préfère me dire que 
- la valeur est une réponse à une question: `Does memberships exist ?` ce qui donne `const doesMembershipExists`
- une affirmation au présent, ce qui donne `const membershipExists`
```suggestion
  const membershipAlreadyExists = await certificationCenterMembershipRepository.isMemberOfCertificationCenter(
```